### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by Shannspan**
Hi! 

I've fixed all the links accross all html pages. 

Also updated each html page to  have correct class ="active" on it's own link.

Also have noticed that we don't appear to have a link from the main sidebar nav to sub-page/accessibility-course.html. Is there a link in another location so the user can access it?

### CODE REVIEW

**Comment by sarahesbie** [comment]
Heya @Shannspan can you have a look at resolving the merge conflict? let us know if you need help.
I think the courses page is linked from the home page, but it could be good to have it linked in the sidebar too!

File: about.html
Comment (sarahesbie) [inline about.html:1]: Heya @Shannspan this is super annoying but can you update all of your links to be like this:
`<a href="accessibility-resources/tools-and-resources.html">`

for some reason GitHub pages has an issue with the preceding slash, ie.e the below won't work:
`<a href="/accessibility-resources/tools-and-resources.html">`

File: index.html
Comment (sarahesbie) [inline index.html:1]: see comment about links above

Comment (sarahesbie) [inline index.html:1]: see comment about links above

File: manifesto.html
Comment (sarahesbie) [inline manifesto.html:1]: see comment about links above

File: sub-page/accessibility-course.html
Comment (sarahesbie) [inline sub-page/accessibility-course.html:1]: see comment about links above

File: sub-page/web-developer-pledge.html
Comment (sarahesbie) [inline sub-page/web-developer-pledge.html:1]: see comment about links above

File: template.html
Comment (sarahesbie) [inline template.html:1]: see comment about links above

File: tools-and-resources.html
Comment (sarahesbie) [inline tools-and-resources.html:1]: see comment about links above

**Review by sarahesbie — CHANGES_REQUESTED** [review]
Heya @Shannspan ! Super annoying link edit request due to github pages!!

**Comment by Shannspan** [comment]
Interesting - will do!

**Comment by Shannspan** [comment]
@sarahesbie - just to check: 

I didn't remove the leading '/' because when I go live from VS code the links don't work without it. 

Do you still want me to go ahead or is more investigation needed?

**Comment by Shannspan** [comment]
Actually - just did a bit of research myself and I can see now that the leading '/' should fix the relative path issue in github. Just confusing in terms of testing the links on the local host! I'll get to it a bit later this afternoon :)

**Comment by Shannspan** [comment]
Hi @sarahesbie the additional commit is because I just realised I hadn't saved changes to the tools-and-resources. html file. All html files should have their nav menu links updated now.

**Review by sarahesbie — APPROVED** [review]

**Comment by sarahesbie** [comment]
@Shannspan looks like you've still got a merge conflict! I'm not at my computer today but can approve PRs from my phone.

**Comment by Shannspan** [comment]
OK - @sarahesbie - I'm really really hoping hat I 'resolved' that correctly! 😬

**Pull request merged**